### ISSUE TITLE
Bug Report: Camera stops bopping when zoom is set to 1.35x or higher

### ISSUE THREAD

**Issue by Average-FNF-Modder** [FunkinCrew/Funkin#3945]
### Issue Checklist

- [x] I have properly named my issue
- [x] I have checked the Issues/Discussions pages to see if my issue has already been reported

### Platform

Newgrounds (Web/HTML5)

### Browser

Microsoft Edge

### Version

0.5.3

### Description (include any images, videos, errors, or crash logs)

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;When using the "Zoom Camera" event, if you set the zoom to 1.25x the stage zoom or above, the camera will stop doing it's a regular bopping. There is also a point on the absolute zoom scale where this happens as well, and while I haven't confirmed it I assume it's whenever that value reaches a zoom above 1.25x the stage zoom as well.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;This effect does not seem to be based on any absolute zoom threshold, but rather specifically on a multiplicative threshold based on the stage, as testing it out on different stages showed the same results for zoom multiplier needed to break the camera's bopping.

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;This bug is pretty annoying, and definitely hinders how creative you can be with your events. Fix it pls!

### Steps to Reproduce

1. Open da game
2. Open chart editor
3. Set your zoom to 1.25x or higher on the relative scale, or to the (what I assume to be) equivalent value on the absolute zoom scale
4. No more bopping :(

**Comment by MrMadera** [comment]
i think it's because of this!
[Image](https://github.com/user-attachments/assets/ab29183a-8c8f-4ad3-bc84-0ae3f672e39e)

i'm not sure if this would be considered an issue or if it's intended,,

**Comment by Average-FNF-Modder** [comment]
> i think it's because of this! ![Image](https://github.com/user-attachments/assets/e3f80e0c-2a10-45d9-9d62-b2b22e6ad7bf)
> 
> i'm not sure if this would be considered an issue or if it's intended,,

If it is intended that's a *really* strange limit that really shouldn't exist lmao.

**Comment by MrMadera** [comment]
> > i think it's because of this! ![Image](https://github.com/user-attachments/assets/e3f80e0c-2a10-45d9-9d62-b2b22e6ad7bf)
> > i'm not sure if this would be considered an issue or if it's intended,,
> 
> If it is intended that's a _really_ strange limit that really shouldn't exist lmao.

i agree! if they decide to implement it, i think it would be better to handle it proportionally, as @cyn0x8 suggested, or simply remove it altogether

**Comment by EliteMasterEric** [comment]
This limit is in place to ensure camera bops don't zoom the camera in too much on high BPMs.

Camera bops should not move the camera closer 35% higher than the current zoom level, however it appears this was implemented relative to the BASE zoom level. So yes, it should be made proportional.

**Comment by Hundrec** [comment]
Hey so Eric got around to fixing this
